doc/pdo.rst

```diff
@@ -31,40 +31,43 @@ Examples
 A :class:`canopen.Node` has a ``.pdo`` attribute that can be used to interact
 with the node using PDOs. This is in turn divided in a ``.tx`` and a ``.rx``
 attribute which can be subindexed to specify which message to use (first map
-starts at 1, not 0)::
+starts at 1, not 0). The :class:`canopen.Node` also allow the user to access 
+the PDOs through the attributes ``.rpdo`` and ``.tpdo`` that provide a more
+direct way to access the configured PDOs (see example below)::
 
     # Read current PDO configuration
-    node.pdo.read()
+    node.pdo[4].read()
```

Comment:
This should be reading both TPDO and RPDO instead.

Reply:
done, calling the ``node.pdo.read()`` again